### PR TITLE
feat: Add RDS cluster activity stream 

### PR DESCRIPTION
**Pull request by magreenbaum**
## Description
Closes: https://github.com/terraform-aws-modules/terraform-aws-rds-aurora/issues/404

## Motivation and Context
https://github.com/terraform-aws-modules/terraform-aws-rds-aurora/issues/404

## Breaking Changes
No.

## How Has This Been Tested?
- [x] I have updated at least one of the `examples/*` to demonstrate and validate my change(s)
- [x] I have tested and validated these changes using one or more of the provided `examples/*` projects

- [x] I have executed `pre-commit run -a` on my pull request

### CODE REVIEW

**Review by bryantbiggs — APPROVED** [review]
Looks great, thank you!

**Comment by antonbabenko** [comment]
This PR is included in version 8.5.0 :tada:

**Comment by Z4ck404** [comment]
Thanks a lot @magreenbaum for the great work

File: main.tf [hunk 1]
```diff
@@ -412,3 +412,18 @@ resource "aws_cloudwatch_log_group" "this" {
 
   tags = var.tags
 }
+
+################################################################################
+# Cluster Activity Stream
+################################################################################
+
+resource "aws_rds_cluster_activity_stream" "this" {
+  count = local.create && var.create_db_cluster_activity_stream ? 1 : 0
```
Comment (Z4ck404) [inline main.tf:421]: There is a limitation on the activity stream as it's not supported by all the instances types.
works only on few of them per this doc : https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/DBActivityStreams.Overview.html 

how is that taken into account ?

Comment (bryantbiggs) [reply main.tf:421]: It's not - that's up to users

**Pull request merged**